Automattic/wp-calypso
# Payments: Add sidebar to credit card screens

#### Changes proposed in this Pull Request

At a high level this PR introduces a new sidebar on the Add/Change credit card screen to help reassure people about entering their information. More specifically, it includes the following updates:

- Add the sidebar to the add new payment method screen.
- Add the sidebar to add/update payment method for a subscription. 
- Replace the credit card loader with a generic payment method loader that includes the sidebar cards. 
- Cleanup the credit card messaging and design on mobile
- Extend the card heading component with a new style which is used in other areas of Calypso but hasn't been codified into the component (I'll clean up the other instances by using this component in another PR).

**Before**
![image](https://user-images.githubusercontent.com/6981253/98825545-9d264e80-2402-11eb-864e-4217fd8b07ff.png)


**After: Add new payment method**
![image](https://user-images.githubusercontent.com/6981253/98823587-4c155b00-2400-11eb-8ec3-11fec5ef471d.png)


**After: Add/update payment method with renew data**
![image](https://user-images.githubusercontent.com/6981253/98823615-55062c80-2400-11eb-8e3c-25cfe5c19077.png)

**After: Add/update payment method without renew data**
![image](https://user-images.githubusercontent.com/6981253/98823649-60595800-2400-11eb-85b2-47d56b36d187.png)


#### Testing instructions


This update should be tested in both the account and site-level billing screens.

* Adding a new payment method from the payment method screen.
* Visit a subscription that has auto-renew enabled and click on update payment method.
* Visit a subscription that doesn't have any payment method associated to it and click on add payment method.
